lib/util/log.sh

```diff
@@ -109,29 +106,28 @@ Log::RegisterLogger() {
 Log::Using() {
   local logger="$1"
   shift
-  if [[ ! -z ${__oo__loggers["$logger"]} ]]
-  then
-      ${__oo__loggers["$logger"]} "$@"
+  if [[ ! -z ${__oo__loggers["$logger"]} ]]; then
```

Comment:
The repository uses syntax with a new line, so I'd prefer to keep it consistent.

Reply:
OK